### ISSUE TITLE
Is there a way to skip headers and/or footers on some pages?

### ISSUE THREAD

**Issue by rohithpr** [Kozea/WeasyPrint#484]
I'm adding common headers and footers to all pages at the moment. If I don't want it to show up on some pages (the first page, for example) I'm generating a separate PDF for that and merging it with the rest of the document.

Is there a better way to skip headers and footers on some pages?

**Comment by chvonrohr** [comment]
I guess this question is asked every few issues here ;)

how I did it was overriding the header for the first page..

```css
@page {
  @top-center {
      background: url('logo.svg') no-repeat 0 0;
      background-size: 10%;
  }
}
@page :first {
  @top-center {
    background: none;
  }
}
```

This is not possible with :last page or any other than the first page. Therefore, it would be awesome to have distinct classes to target pages. But, I guess there is no "CSS standard" for that and thus it's not integrated in the core...

**Comment by liZe** [comment]
Some of these problems could be solved with named pages, see #57.
